xmlWorksheet.go

```diff
@@ -204,6 +204,7 @@ type xlsxSheetView struct {
 	WorkbookViewID           int              `xml:"workbookViewId,attr"`
 	Pane                     *xlsxPane        `xml:"pane,omitempty"`
 	Selection                []*xlsxSelection `xml:"selection"`
+	ShowRuler                *bool            `xml:"showRuler,attr,omitempty"`
```

Comment:
Newly added field `showRuler` should be following order in [OOXML Spec](https://github.com/sc34wg4/OOXMLSchemas/blob/5d7db375ff5f5ecfd38d39d088a6b81a6c41e39e/Part1/OfficeOpenXML-XMLSchema-Strict/sml.xsd#L2315-L2341) to avoid generating file corrupted with compatibility issue.

Reply:
Done!